bio-seq/src/codec/iupac.rs

```diff
@@ -54,7 +54,26 @@
 //! ```
 use crate::codec::{dna::Dna, Codec};
 use crate::seq::{Seq, SeqArray, SeqSlice};
-//use crate::{Complement, Reverse, ReverseComplement};
+use crate::{Complement, ComplementMut};
+
+// const IUPAC_COMPLEMENT_TABLE: [u8; 256] = [
```

Comment:
This is good, but we just need 16 items:
```
[u8; 16] = [0b0000, 0b1000, 0b0100, ..., 0b1111];
```
or something like this. And this way won't need to shift the result of the lookup.